Python/Product/Cookiecutter/Model/CookiecutterClient.cs

```diff
@@ -71,15 +71,73 @@ class CookiecutterClient : ICookiecutterClient {
 
         public async Task CreateCookiecutterEnv() {
             // Create a virtual environment using the global interpreter
-            var interpreterPath = _interpreter.InterpreterExecutablePath;
-            var output = ProcessOutput.RunHiddenAndCapture(interpreterPath, "-m", "venv", _envFolderPath);
+            try {
+                await CreateVenv();
+            } catch (ProcessException ex) when (ex.Result.ExitCode == 1) {
+                // Create fails on some Anaconda due to venv failing to install pip.
+                // Try again by installing pip ourselves.
+                await CreateVenvWithoutPipThenInstallPip();
+            }
+        }
 
-            await WaitForOutput(interpreterPath, output);
+        private async Task CreateVenv() {
+            if (Directory.Exists(_envFolderPath)) {
```

Comment:
Directory may be deleted right after you've checked for it. It is more reliable to just call `Directory.Delete` and catch `DirectoryNotFoundException`.

Reply:
I still want to check it exists because I only want to show the message if we'll try to delete it. I'll add the catch so that we don't fail if it no longer exists by the time we delete.

Reply:
Ok